go/chat/storage/blockengine_memcache.go

```diff
@@ -35,15 +35,14 @@ func (c *logContext) GetVDebugLog() *libkb.VDebugLog {
 type blockEngineMemCacheImpl struct {
 	blockCache *lru.Cache
 	logContext *logContext
-	lockTab    *libkb.LockTable
+	lockTab    libkb.LockTable
```

Comment:
What is this for?

Reply:
just cleaning up how the other packages use this

Reply:
How is this even correct? All of `LockTable`'s methods have pointer receivers